styles.css

```diff
@@ -326,17 +298,13 @@ h2.project-title {
   padding: 10px 12px 10px 12px;
 }
 
-.used-tools > li:last-child {
-  border: none;
-}
-
 .see-prj {
-  width: 89px;
-  height: 48px;
   background-color: #7d73fc;
   color: #ecebff;
-  margin-left: 30px;
+  width: 89px;
+  height: 48px;
   border: none;
+  order: 3;
 }
```

Comment:
The button is not placed correctly in your design.

Your design:

<img width="175" alt="Screen Shot 2022-06-07 at 12 27 45" src="https://user-images.githubusercontent.com/82426682/172445557-25b08b95-c933-4a97-b355-f038ed23ca07.png">

Template design:

<img width="213" alt="Screen Shot 2022-06-07 at 12 28 11" src="https://user-images.githubusercontent.com/82426682/172445629-bd1905f6-ceee-4a5b-a204-835d7efcea4a.png">

Please fix this as it is a project requirement.

<img width="897" alt="Screen Shot 2022-06-07 at 12 28 35" src="https://user-images.githubusercontent.com/82426682/172445681-133e8366-6529-4521-8e77-fb9b63d9f5d8.png">